js/languagePickerModel.js

```diff
@@ -9,6 +9,9 @@ define([
       "_isEnabled": false,
       "displayTitle": "",
       "body": "",
+      "_defaultLanguage": "en",
+      "_defaultDirection": "ltr",
```

Comment:
sorry, just realised I'm duplicating Tom's comment...!

aside from this it's great, just tested and it works well.